pkg/scalers/gcp_pubsub_scaler.go

```diff
@@ -145,14 +145,14 @@ func (s *pubsubScaler) IsActive(ctx context.Context) (bool, error) {
 			s.logger.Error(err, "error getting Active Status")
 			return false, err
 		}
-		return size > s.metadata.activationValue, nil
+		return size > float64(s.metadata.activationValue), nil
```

Comment:
if `value` is `float64`, the `activationValue` should be also `float64`, please update the type in the definition instead of casting the type here (updating the parsing here and the docs)

Reply:
The metadata parsing has been updated and removed the cast to float64.